docs/mkdocs.yml

```diff
@@ -120,6 +120,7 @@ nav:
           - Using Postmodeling: postmodeling/index.md
           - Postmodeling & Crosstabs Configuration: postmodeling/postmodeling-config.md
       - Model governance:  dirtyduck/ml_governance.md
+      -Predictlist: predictlist/index.md
```

Comment:
Should this be `- Predictlist:` (I can't remember if YAML caries about the whitespace between the `-` and list item)?